src/components/Catalog.tsx

```diff
@@ -106,17 +106,19 @@ export const Catalog = ({ handleClose }: { handleClose: () => void }) => {
             <>
               <ToolbarItem style={{ padding: '0', marginRight: '0' }}>
                 <InputGroup>
-                  <SearchInput
-                    name={'stepSearch'}
-                    id={'stepSearch'}
-                    type={'search'}
-                    placeholder={'search for a step...'}
-                    aria-label={'search for a step'}
-                    value={query}
-                    onChange={(_e, s) => changeSearch(s)}
-                    onClear={() => setQuery('')}
-                    ref={searchInputRef}
-                  />
+                  <div className={'catalog__searchInputWrapper'}>
```

Comment:
TBH I tried to figure out how to do it the way you are considering, but in this case the behaviour - meaning the "expanding" search field in case I typed some input remained unchanged, so I chose this approach.